weldx/tests/asdf_tests/test_asdf_util.py

```diff
@@ -186,3 +188,46 @@ def test_get_highest_tag_version():
 def test_get_schema_tree():
     d = get_schema_tree("single_pass_weld-0.1.0")
     assert isinstance(d, dict)
+
+
+class TestProtectedView(unittest.TestCase):
+    def setUp(self):
+        data = dict(foo="blub", bar=42)
+        self.protected_key = "bar"
+        self.view = _ProtectedViewDict(protected_keys=[self.protected_key], data=data)
+
+    def test_protected_keys_hidden(self):
+        assert self.protected_key not in self.view.keys()
+        assert self.protected_key not in self.view
+        assert (self.protected_key, "bar") not in self.view.items()
```

Comment:
```suggestion
        assert (self.protected_key, 42) not in self.view.items()
```
it probably doesn't matter but should this be the key/value item pair?

Reply:
you're absolutely right :+1: